liota/dccs/iotcc.py

```diff
@@ -400,7 +412,7 @@ def write_entity_json_file(self, prop_dict, attribute_list, uuid, remove):
         if prop_dict is not None:
             for key in prop_dict.iterkeys():
                 value = prop_dict[key]
-                if key == 'entity type' or key == 'name' or key == 'device type':
+                if key == 'entity type' or key == 'name' or key == 'device type' or key == 'Entity_Timestamp':
```

Comment:
what is the reason for this change?

Reply:
This fun writes uuid.json file under devs folder (configurable) for AW to take and remove. It is based on local record bsed on entity folder (configurable). In this pull request, I added Entity_Timestamp to assist judgement of restarting, which is needed in local records. But it is not needed for uuid.json file for AW. Therefore, has to skip it.